lib/galaxy/managers/configuration.py

```diff
@@ -108,6 +108,7 @@ def _use_config(config, key, **context):
             'file_sources_configured'           : lambda config, key, **context: self.app.file_sources.custom_sources_configured,
             'upload_from_form_button'           : _use_config,
             'release_doc_base_url'              : _use_config,
+            'user_library_import_dir_available' : lambda config, key, **context: True if config.get('user_library_import_dir') else False,
```

Comment:
```suggestion
            'user_library_import_dir_available' : lambda config, key, **context: bool(config.get('user_library_import_dir')),
```
that's a little simpler and what we're doing for `server_mail_configured ` above.

Reply:
thanks